cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfParser.h

```diff
@@ -153,7 +155,7 @@ struct UrdfRobotDesc
     UrdfLinkDesc* root_link_desc_ = nullptr;
 };
 
-class UrdfParser
+class URDFBOT_API UrdfParser
```

Comment:
I don't think we need to expose UrdfParser outside of this module. I think the only thing we need to expose are the pawn and component classes.

Reply:
fixed.